custom_components/opensprinkler/const.py

```diff
@@ -12,6 +12,7 @@
 DOMAIN = "opensprinkler"
 
 DEFAULT_NAME = "OpenSprinkler"
+DEFAULT_VERIFY_SSL = True
```

Comment:
Perhaps better to default to false

Reply:
Are you thinking the default state should be allowing non trusted certificates? Just double checking.